Makefile

```diff
@@ -16,7 +16,7 @@ help: ## Display this help text
 	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
 install: ## Install dependencies locally
-	make --jobs=2 $(MAKEFLAGS) node_modules gitmodules
+	$(MAKE) node_modules gitmodules
```

Comment:
`$(MAKE)` necessary to pass the `MAKEFLAGS`, otherwise it evaluates to just `MAKEFLAGS=w` rather than `MAKEFLAGS=w -j --jobserver-fds=3,4`.

Reply:
Since the jobserver is the same and has a total, the number of jobs is unnecessary. The subtargets are executed in parallel (between those two, but also with everything else):
```
make[2]: Entering directory '/home/giorgio/code/article-store'
npm install
...
git submodule update --init --recursive
...
touch node_modules
```

Reply:
When coming from `make dev` rather than `make install`?

Reply:
Rather speed up `dev` than `install`, so let's go with this. Though these should now be prerequisites?

Reply:
Yes, `make install` doesn't break because there are no preset `MAKEFLAGS` coming from a parent `make --jobs` when invoked (I think).

Could be prerequisites, but making a minimal change to fix `master` first.